pinot-segment-local/src/main/java/org/apache/pinot/segment/local/utils/nativefst/ImmutableFST.java

```diff
@@ -139,7 +140,11 @@ public final class ImmutableFST extends FST {
    * see the documentation of this class for more information on how this
    * structure is organized.
    */
-  public final OffHeapMutableBytesStore _mutableBytesStore;
+  //public final OffHeapMutableBytesStore _mutableBytesStore;
+
+  public final PinotDataBuffer _pinotDataBuffer;
```

Comment:
@Jackie-Jiang I think I need to use the offset buffer and value buffer approach here instead of incrementing the offset by PER_BUFFER_SIZE each time. Is that correct? (wanted to confirm with you before I make the change)